docs/source/calling-smart-contracts.md

```diff
@@ -0,0 +1,125 @@
+# Calling smart contracts
+Currently, there are 2 different ways to call another smart contract on the blockchain: using a `call_contract` method 
+or using an interface, but internally they work both the same way.
+
+Let us suppose the following contract was deployed on Neo:
+```python
+from boa3.builtin.compile_time import public
+
+
+@public
+def hello_stranger(name: str) -> str:               
+    return "Hello " + name
+```
+To call the `hello_stranger` method, first you'll need to know the smart contract's [script hash](https://developers.neo.org/docs/n3/develop/deploy/deploy#the-contract-scripthash),
+Let us also assume it is `0x000102030405060708090A0B0C0D0E0F10111213`.
+
+## with call_contract
+Use the `call_contract` method from `boa3.builtin.interop.contract` on your smart contract. You'll need to use the 
+script hash, followed by the name of the function you want to call, followed by the arguments of said function. You'll 
+also need to type cast the return, otherwise it will be considered as `Any`.
```

Comment:
You'll also need to type cast the return **so the compiler type checker works as expected**, otherwise the return type will be considered as `Any`